gammapy/data/counts_spectrum.py

```diff
@@ -0,0 +1,188 @@
+
+import numpy as np
+from astropy import log
+from ..spectrum.energy import *
+import datetime
+from astropy.io import fits
+
+__all__ = ['CountsSpectrum']
+
+class CountsSpectrum(object):
+    """Counts spectrum dataset
+
+    Parameters
+    ----------
+
+    counts: `~numpy.array`, list
+        Counts
+    energy: `~gammapy.spectrum.Energy`, `~gammapy.spectrum.EnergyBounds`.
+        Energy axis
+
+    Examples
+    --------
+
+    .. code-block:: python
+    
+        from gammapy.spectrum import *
+        from gammapy.data import *
+        ebounds = EnergyBounds.equal_log_spacing(1,10,10,'TeV') 
+        counts = [6,3,8,4,9,5,9,5,5,1]
+        spec = CountsSpectrum(counts, ebounds)
+        hdu = spec.to_pha() 
+    """
+
+    def __init__(self, counts, energy, livetime=None):
+        
+        if np.asarray(counts).size != energy.nbins:
+            raise ValueError("Dimension of {0} and {1} do not"
+                             " match".format(counts, energy))
+        
+        if not isinstance(energy, Energy):
+            raise ValueError("energy must be an Energy object")
+        
+        self.counts = np.asarray(counts)
+
+        if isinstance (energy, EnergyBounds):
+            self.energy = energy.log_centers
+        else:
+            self.energy = energy
+
+        self._livetime = livetime
+
+    @property
+    def entries(self):
+        """Total number of counts
+        """
+        return self.counts.sum()
+
+    @property
+    def livetime(self):
+        """Live time of the dataset
+        """
+        return self._livetime
+
+    @classmethod
+    def from_fits(cls, hdu):
+        """Read SPECTRUM fits extension from pha file (`CountsSpectrum`).
+
+        Parameters
+        ----------
+        hdu: `~astropy.io.fits.BinTableHDU`
+            ``SPECTRUM`` extensions.
+        """
+        pass
+
+    @classmethod
+    def from_eventlist(cls, event_list, bins):
+        """Create CountsSpectrum from fits 'EVENTS' extension (`CountsSpectrum`).
+
+        Subsets of the event list should be choosen via the approriate methods
+        in `~gammapy.data.EventList`.
+
+        Parameters
+        ----------
+        event_list: 
+            `~astropy.io.fits.BinTableHDU, `gammapy.data.EventListDataSet`,
+            `gammapy.data.EventList`, str (filename)
+        bins: `gammapy.spectrum.energy.EnergyBounds`
+            Energy bin edges
+        """
+
+        if isinstance (event_list,fits.BinTableHDU):
+            event_list = EventList.read(event_list)
+        elif isinstance(event_list, gammapy.data.EventListDataSet):
+            event_list = event_list.event_list
+        elif isinstance(event_list, str):
+            event_list = EventList.read(event_list, hdu='EVENTS') 
+
+        energy = event_list.energy.to(bins.unit)
+        val = np.histogram(energy,bins)
+        livetime = event_list.observation_live_time_duration
+
+        return cls(val, energy, livetime)
+
+
+    def to_pha(self, **kwargs):
```

Comment:
Is kwargs needed here?
If yes, mention in the docstring what the user can pass in?